todo-mvc-kitchensink/tests/functional/Page.ts

```diff
@@ -79,7 +83,10 @@ export default class Page {
 			.get('http://localhost:9000/_build/src/index.html')
 			.setFindTimeout(5000)
 			.findByCssSelector(this.selectors.newInput)
-			.setFindTimeout(100);
+			.setFindTimeout(100)
+			.then(() => {
+				return this.errorCollector.init()
```

Comment:
we have a cli linting repo but I don’t think we’ve done anything with it yet, https://github.com/dojo/cli-lint-tslint